src/PowerShell/tests/Microsoft.WinGet.DSC.Tests.ps1

```diff
@@ -242,3 +242,36 @@ Describe 'WinGetPackageManager' {
 
     # TODO: Add test to verify Set method for WinGetPackageManager
 }
+
+Describe 'WinGetConfigRoot' {
+    It 'Sets and removes $WinGetConfigRoot by scope' -ForEach 'User','Machine' {
+        $get = InvokeWinGetDSC -Name WinGetConfigRoot -Method Get -Property @{ WinGetScope = $_ }
+        $get.WinGetConfigRoot = "WinGetConfigRoot"
+        $get.Exist | Should -Be $false
+        $get.Path | Should -Be ""
+        $get.WinGetScope | Should -Be $_
+
+        $test = InvokeWinGetDSC -Name WinGetConfigRoot -Method Test -Property @{ Exist = $true; Path = 'C:\Foo\Bar'; WinGetScope = $_ }
+        $test.InDesiredState | Should -Be $false
+
+        InvokeWinGetDSC -Name WinGetConfigRoot -Method Set -Property @{ Exist = $true; Path = 'C:\Foo\Bar'; WinGetScope = $_ }
+
+        # Verify that $WinGetConfigRoot variable is set.
+        $result = InvokeWinGetDSC -Name WinGetConfigRoot -Method Get -Property @{WinGetScope = $_}
+        $result.Exist | Should -Be $true
+        $result.Path | Should -Be 'C:\Foo\Bar'
+        $result.WinGetScope | Should -Be $_
+
+        # Remove $WinGetConfigRoot variable.

```

Comment:
Before you remove it, it may be a good idea to check that if a different Path is specified while the variable exists, then the test returns false.